### PR TITLE
Arcgis webpack angular branch

### PR DESCRIPTION
**Pull request by andygup**
@araedavis I think I got everything in here. Can you please take a look at the README and code as another set of eyes?

### CODE REVIEW

**Comment by andygup** [comment]
TO-DO: update the master branch README with this new branch name.

File: README.md [hunk 1]
```diff
@@ -1,8 +1,6 @@
 # angular-cli-esri-map
 
-This branch contains a simple but complete application that uses the ArcGIS API for JavaScript, an enterprise geospatial API, and Angular CLI. It uses esri-loader, a small library that helps load ArcGIS JavaScript API modules (v3.x or v4.x) in non-Dojo applications.
-
-To use arcgis-webpack-plugin instead of esri-loader to load the ArcGIS API for JavaScript modules, check out the `@TODO ADD BRANCH NAME` branch of this repo.
+This branch contains a simple but complete application that uses the ArcGIS API for JavaScript, an enterprise geospatial API, and Angular CLI. It uses arcgis-webpack-plugin to help load ArcGIS JavaScript API modules (v3.x or v4.x) in ArcGIS applications.
 
 If you would like to create your own Angular CLI project from scratch and incorporate these components, [create a new Angular CLI project](https://cli.angular.io/) and copy the `src/app/` directory of this repo to your new project. You will need to install `esri-loader` and `@types/arcgis-js-api` manually.
```
Comment (araedavis) [inline README.md]: You will need to install `arcgis-webpack-plugin`

Comment (andygup) [reply README.md]: Yes, good catch. I'll make the update.

Comment (andygup) [reply README.md]: I'll also add a reference to the custom webpack builders plugin...forgot that too,

**Comment by andygup** [comment]
Sweet, I'll merge then this do a quick PR to update the `main` branch README with this branch name.

**Pull request merged**